dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WebContainerRequestState.java

```diff
@@ -231,19 +230,35 @@ public void setCookieAttributes(String cookieName, String cookieAttributes) {
             logger.logp(Level.FINE, CLASS_NAME, methodName, " cookieName --> " + cookieName + " cookieAttribute --> " + cookieAttributes);
         }
 
-        //future cookieAttributes can be further separated with semicolon delimiter:  attributeName=attributeValue;attributeName2=attributeValue2;singleAttributeNameNoValue
-        //Currently ignore all but SameSite 
         String[] attribute = cookieAttributes.split("=");
-        if (!attribute[0].equals("SameSite")) {
+        /*
+         * Beta: This method will start accepting "Parititioned" as a new attribute. It will need to be in the form "partitioned=boolean".
+         */
```

Comment:
It needs to be `partitioned=true/false` is a bit more clear IMO

Reply:
I can update it